app/SharedWebComponents/Services/IPdfViewer.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SharedWebComponents.Services;
+
+public interface IPdfViewer
+{
+    Task ShowDocument(string name, string url);
```

Comment:
Let's use the `Async` suffix for consistency. Also, consider the `ValueTask` as the return type to help avoid allocations if possible.

```suggestion
    ValueTask ShowDocumentAsync(string name, string url);
```

Reply:
Updated